coremltools/converters/nnssa/coreml/ssa_converter.py

```diff
@@ -731,6 +732,126 @@ def _convert_custom_layer(self, node):
             raise ValueError('Custom Shape Function for {} not provided!'.format(node.op))
         shapes.propagate_single_layer(layer, self.tensor_shapes, custom_shape_function=self.custom_shape_functions[node.op])
 
+    def _convert_einsum(self, node):
+
+        input_nodes, input_names, input_types = self._get_input_tensors(node)
+        if len(input_names) > 2:
+            raise ValueError('current einsum only supports inputs tensors number <= 2.')
```

Comment:
is there a case when number of inputs are more than 2? 

change error message to `currently, 'einsum' operation is only supported when it has less than equal to 2 inputs.`

Reply:
Yes :) the number of the inputs could be as many as you like,
so the equation will be something like 'ijk,ijk,ijkt->t', as long as it can be written in a summation format.
 Will change the error message :)